student-work/2022/Debbie-Mankowitz/Glossary of FinTech Technology

```diff
@@ -0,0 +1,99 @@
+                		**Glossary of FinTech Technology**
```

Comment:
This contribution is in the wrong location. 
Please make a pull request here - https://github.com/Laura-Novich-OBW/glossary and add a fintech directory. 
please close this PR create a new one in the correct location and address the other comments as well.